ext-src/packages/maven/MavenUtils.ts

```diff
@@ -117,6 +122,10 @@ export class MavenUtils {
     });
 
     // TODO: The dependency list brought back appears to have a ton of duplicates, it needs to be deduped at a minimum in the future
+    //   we added dependencyListString above as a brute force way to dedupe. there is probably a better way to dedupe, but we couldn't 
```

Comment:
is the `TODO` above still required?

Reply:
strictly speaking, no. dependencyList will now be unique. however, as my additional commentary explains, this being my first experience with typescript, i did not see an easy way to deduplicate an array of MavenPackage objects. I assume we would need a custom transform. And given that it is probably better to do this closer to the IQ API calls so that the deduplication applies to all formats, and does not need to be implemented for each one individually, i'd prefer to leave the todo here.